### ISSUE TITLE
Definition of Federal information and Federal information system

### ISSUE THREAD

**Issue by djharrity** [ombegov/a130#22]
Line 941 define Federal information as "by another organization on behalf of an agency."  

Many agencies may have grants management activities that are provide partial or full funding of a website funded by cooperative agreements and not contracts. As such, these programs often if not always, are funded by cooperative agreements may not go through the 
a) Exhibit 300 or Exhibt 53 process; (
b) FISMA process, 
c) are hosted as non.-gov domains (using .com, .org or other gTLDs) and 
d) often include disclaimer statements on the website that the "Views expressed on this website in whole or in part, do not necessarily reflect  the views or policies of {{Agency name}}."

Together these factors, agencies often do not consider these websites or IT systems supporting grants via cooperative agreements to be Federal information and Federal information systems.

Clarity within this document regarding grant management activities and cooperative agreements appears to be needed.

**Comment by NoahKunin** [comment]
@djharrity Great point.

This is part of the currently accepted definition of a Federal information system as well. A-130 doesn't change this - current policy is that basically anything an agency does to cause an information system to exist (whether it includes :moneybag: or not) means it is within scope of the entire IT policy and compliance infrastructure - that same definition puts anything we even _use_ also in that scope.

In practice, this isn't enforced or measured in any meaningful way. For example, many agencies just automatically put "out of scope" many third party systems that are not relevant to their core mission. Agency use of Twitter or Facebook for example usually do not go through a FISMA process, aren't on a .gov, etc. Because even processing a simple risk management decision through current policies of those systems would be overly cost and time prohibitive, agencies instead keep their use "off the books".

The A-130 site provided by OMB, is itself, a good example of this. It uses CloudFlare, the GitHub Pages infrastructure on .io, and GitHub itself, none of the above having gone through any meaningful FISMA review. CloudFlare doesn't even have a Federal Terms of Service in place. It sort of brings "do as I say, not as I do" to a _whole_ new level. 

This is a good opportunity for OMB to more properly scope A-130, so agencies can focus what really matters, and bring their current "unapproved" usage of information system out into the :sun_with_face:. Keeping usage in the shadows helps no one, and also prevents and disincentives clear and open communication on what best practices we **should** be using with these third-party services.

**Comment by thecapacity** [comment]
I agree with @NoahKunin that :sun_with_face: is important - and removing some of the "threat" associated with some of the legitimate uses (e.g. github, cloudflare) - which don't meet every single mandate - is important for us all to collectively figure out what mandates are important.

e.g. I once had someone tell me "Open Source is nice, but the Government doesn't use much of it..." to which I asked why then there was a sudden need to scramble for heartbleed...
